submission/server/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/google/certificate-transparency-go/submission"
+)
+
+// Flags.
+var (
+	httpEndpoint           = flag.String("http_endpoint", "localhost:5951", "Endpoint for HTTP (host:port)")
+	logListPath            = flag.String("loglist_path", "https://www.gstatic.com/ct/log_list/log_list.json", "Path for list of CT Logs in JSON format")
+	logListRefreshInterval = flag.Duration("loglist_refresh_interval", 2*24*time.Hour, "Interval between consecutive reads of Log-list")
```

Comment:
supernit: Why 2 days? Seems like something along 1 hour could also be a reasonable default. Imagine you are the developer updating the log list. It's kinda sad if you need to wait 2 days until your update propagates.

Reply:
As some default. Likely to get configured/adjusted.

Reply:
supernit: If it's going to be changed anyway then 1 day seems like a nicer default than 2 days, as it doesn't trigger "why 2?" questions.

Reply:
)